### PR TITLE
optimize: optimize some scripts related to Apollo

### PR DESCRIPTION
**Pull request by wangliang181230**
optimize: optimize some scripts related to Apollo
优化一些与 Apollo 相关的脚本。

### CODE REVIEW

File: script/client/conf/registry.conf [hunk 1]
```diff
@@ -95,6 +95,7 @@ config {
     apolloMeta = "http://192.168.1.204:8801"
     namespace = "application"
     apolloAccesskeySecret = ""
+    cluster = "seata"
```
Comment (funky-eyes) [inline script/client/conf/registry.conf:98]: This did not previously have a default value and it is suggested that it be changed to `""`.

File: script/client/spring/application.properties [hunk 2]
```diff
@@ -90,6 +90,7 @@ seata.config.apollo.apollo-meta=http://192.168.1.204:8801
 seata.config.apollo.apollo-accesskey-secret=
 seata.config.apollo.app-id=seata-server
 seata.config.apollo.namespace=application
+seata.config.apollo.cluster=seata
```
Comment (funky-eyes) [inline script/client/spring/application.properties:93]: ibid

File: script/client/spring/application.yml [hunk 3]
```diff
@@ -84,6 +84,7 @@ seata:
       app-id: seata-server
       namespace: application
       apollo-accesskey-secret:
+      cluster: seata
```
Comment (funky-eyes) [inline script/client/spring/application.yml:87]: ibid

File: script/client/conf/registry.conf (hunk 1 above)
Comment (wangliang181230) [reply script/client/conf/registry.conf]: done

**Comment by wangliang181230** [comment]
@funky-eyes @slievrly LGTM

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**